src/applications/personalization/view-dependents/components/ViewDependentsSidebar/ViewDependentsSidebarBlockStates/ViewDependentSidebarBlockStates.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export const firstSidebarBlock = {
+  heading: 'When do I need to declare or modify a dependent?',
+  content: (
+    <>
+      <p>
+        You need to let VA know when there is a{' '}
+        <strong>change in a dependent's status.</strong>
+        Changes in status could include:
+      </p>
+      <ul>
+        <li>The birth or adoption of a child</li>
+        <li>If you get married or divorced</li>
+        <li>If a child becomes seriously disabled</li>
+        <li>If your child is over 18 and is not attending school</li>
+      </ul>
+    </>
+  ),
+};
+
+export const secondSidebarBlock = {
+  heading: 'What if I need help with my claim?',
+  content: (
+    <>
+      <p>
+        You can work with an accredited Veterans Service Officer (VSO). We trust
+        these professionals because they are trained and certified in the VA
+        claims and appeals process. A VSO can answer your questions or even file
+        a claim for you.
+      </p>
+      <a
+        href="#"
+        aria-label="Get help filing a claim"
+        title="Get help filiing a claim"
+      >
+        Get help filing a claim
+      </a>
+    </>
+  ),
+};
+
+export const thirdSidebarBlock = {
+  heading: 'What if I have questions?',
+  content: (
+    <>
+      <p>
+        You can call us at <a href="tel:8008271000">800-827-1000</a>. We're here
```

Comment:
This telephone link could be updated with `aria-label="800. 8 2 7. 1 0 0 0"`. This was touched on in a recent accessibility PR - https://github.com/department-of-veterans-affairs/vets-website/pull/11267#discussion_r356354007